src/app/examples/list/list-example.component.html

```diff
@@ -57,6 +57,45 @@ <h2>List with selectable items:</h2>
 
 <hr />
 
+<h2>List with colored items:</h2>
+<kirby-list [items]="items" [itemColorCb]="itemColorCb" (itemSelect)="onItemSelect($event)" showDivider="true">
+  <kirby-list-item
+    *kirbyListItem="let item"
+    [item]="item"
+    [title]="item.title"
+    [amount]="item.amount"
+  ></kirby-list-item>
+</kirby-list>
+
+ <hr />
+
+ <h2>List with sections and colored items:</h2>
+  <kirby-list
+    [items]="items"
+    (itemSelect)="onItemSelect($event)"
+    [getSectionName]="getSectionName"
+    [itemColorCb]="itemColorCb"
+    [showDivider]="true">
+  <!-- kirby-list-header is not supported in conjunction with *kirbyListSectionHeader due to a bug in RadListView on iOS -->
```

Comment:
This comment is no longer relevant. Please delete it. And if you copied from somewhere, please also delete it there.